diff -Naur Stupid_old/simple/llcapi/Makefile Stupid_new/simple/llcapi/Makefile
--- Stupid_old/simple/llcapi/Makefile	2016-12-03 21:51:16.393018245 +0100
+++ Stupid_new/simple/llcapi/Makefile	2016-12-03 21:54:07.035142032 +0100
@@ -1,10 +1,10 @@
 EXES = ctest
 SLIB = ../../lib/libllcapi.so
 
-CXX := clang++-3.8
+CXX := clang++
 RTTIFLAG := -fno-rtti
-CXXFLAGS := $(shell llvm-config-3.8 --cxxflags) $(RTTIFLAG)
-LLVMLDFLAGS := $(shell llvm-config-3.8 --ldflags --libs) -ldl
+CXXFLAGS := $(shell llvm-config --cxxflags) $(RTTIFLAG)
+LLVMLDFLAGS := $(shell llvm-config --ldflags --libs) -ldl
 
 OBJECTS = main.o
 
diff -Naur Stupid_old/simple/llib/Makefile Stupid_new/simple/llib/Makefile
--- Stupid_old/simple/llib/Makefile	2016-12-03 21:51:16.393018245 +0100
+++ Stupid_new/simple/llib/Makefile	2016-12-03 22:08:53.029892794 +0100
@@ -1,5 +1,5 @@
 simpledef.ll: simpledef.c
-	clang-3.8 -S -O3 --target=i386-- -emit-llvm simpledef.c -o tmp.ll
+	clang -S -O3 --target=i386-- -emit-llvm simpledef.c -o tmp.ll
 	sed -r -f ll.sed tmp.ll >simpledef.ll
 	rm tmp.ll
 
diff -Naur Stupid_old/test/Makefile Stupid_new/test/Makefile
--- Stupid_old/test/Makefile	2016-12-03 21:51:16.393018245 +0100
+++ Stupid_new/test/Makefile	2016-12-03 21:58:01.793625496 +0100
@@ -35,7 +35,7 @@
 $(ISF).mac.ll: $(ISF).stp ../Simple
 	../Simple $(ISF).stp i
 test.s: test.ll
-	llc-3.8 -relocation-model=pic -O3 test.ll -o test.s
+	llc -relocation-model=pic -O3 test.ll -o test.s
 else
 all: $(TGTS)
 # pro cortex-m4f s floating point unit
@@ -58,11 +58,11 @@
 $(ISF).mac.ll: $(ISF).stp ../Simple
 	../Simple $(ISF).stp f
 test.s: test.ll
-	llc-3.8 -function-sections -O3 $(LLCFP) test.ll -o test.s
+	llc -function-sections -O3 $(LLCFP) test.ll -o test.s
 endif
 
 test.ll: $(ISF).mac.ll
-	opt-3.8 -S -O3 $(ISF).mac.ll -o test.ll
+	opt -S -O3 $(ISF).mac.ll -o test.ll
 
 %.o: %.s
 	$(AS)  $< -o $@
